cypress/integration/actions_spec.js

```diff
@@ -0,0 +1,21 @@
+import {actions} from '../../src/actions'
+
+describe('actions', () => {
+  context('add', () => {
+    it('adds an item', () => {
+      const state = {
+        input: 'foo',
+        todos: []
+      }
+      const result = actions.add()(state)
+      cy.wrap(result).should('deep.equal', {
```

Comment:
Out of curiousity, why `cy.wrap` over `expect`? It looks like cypress bundles with chai's expect, so I'm curious if it's the convention of cypress, support for cy.wrap is better, or just personal preference.

Reply:
same thing, I changed it to simple `expect`, no preference